ui/dashboards/src/components/Variables/TemplateVariable.tsx

```diff
@@ -104,7 +104,7 @@ function ListVariable({ name }: TemplateVariableProps) {
         <Select
           sx={{ minWidth: 100, maxWidth: 250 }}
           id={name}
-          label={name}
+          label={title}
```

Comment:
This is a really subtle UI bug that seemed like it might not really be worth setting up visual tests for. Let me know if you disagree, and I'll add some.